submissions/morenko/building-a-tiny-js-world/index.js

```diff
@@ -0,0 +1,43 @@
+const woman = {
+    species: 'human',
+    name: 'line',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: 'bye',
+  };
```

Comment:
I'm going to write this once, but you have to fix indentations in the whole file.